src/main/java/uk/co/ramp/covid/simulation/place/CommunalPlace.java

```diff
@@ -70,6 +72,27 @@ public void moveShifts(Time t, boolean lockdown) {
         }
         people.removeAll(left);
     }
+
+
+    /** Moves Phase2 people to either hospital or back home */
+    public void movePhase2(Time t, Places places) {
+        List<Person> left = new ArrayList<>();
+        for (Person p : people) {
+            if (p.cStatus() != null && p.cStatus() == CStatus.PHASE2) {
+                if (p.goesToHosptialInPhase2()) {
+                    Hospital h = places.getRandomHospital();
```

Comment:
Should there be a limit on the number of patients a hospital can accept?

Reply:
> Should there be a limit on the number of patients a hospital can accept?

Hi Bob. In the short term let's not worry about this, it hasn't been an issue because of the selection of cases for hospitlaisation in Scotland, as we are modelling this rate, we need not woryr about this just now